mobile_launch.json

```diff
@@ -1,5 +1,5 @@
 {
-  "template": "small",
+  "template": "small_entity_db_v2",
```

Comment:
was that intentional?

Reply:
Yes, this is a V2 template that supports KCP. Old one didn't allow KCP connection